lib/tasks/avalon.rake

```diff
@@ -223,6 +223,16 @@ EOC
         end
       end
     end
+
+    desc "Starts Status Checking and Email Notification of Existing Batches"
+    task :ingest_status_check => :environment do
+      IngestFinished.perform_later
+    end
+
+    desc "Status Checking and Email Notification for Stalled Batches"
+    task :ingest_stalled_check => :environment do
+      StalledJob.perform_later
```

Comment:
If this is run once a day then I don't think we need to worry about multiple of the jobs running at the same time or we'd have some other pretty major issues.  But maybe if they were given a low priority it might be possible to happen?

Reply:
I'm not quite clear on the proposed change/fix here

Reply:
I think this is okay for now unless we wanted to be consistent across the three tasks.  I think there is still a risk to double running things albeit very very low.